### PR TITLE
Add dialog to create a Scheduled Report

### PR DESCRIPTION
**Pull request by ericpgreen2**
This PR creates a dialog to create a Scheduled Report.

Follow-on work:
- Time zone selection
- Make the "Recipients" combobox accessible

### CODE REVIEW

File: web-admin/src/routes/+layout.svelte
Comment (djbarnwal) [inline web-admin/src/routes/+layout.svelte:37]: Can you add some comment around it.

File: web-common/src/components/forms/DatePicker.svelte
Comment (djbarnwal) [inline web-common/src/components/forms/DatePicker.svelte:15]: Any we not using the custom date picker we have in our time range?

File: web-common/src/components/menu-v2/MenuItem.svelte
Comment (djbarnwal) [inline web-common/src/components/menu-v2/MenuItem.svelte:4]: Would be good to have some documentation around these

File: web-common/src/features/dashboards/dimension-table/ExportDimensionTableDataButton.svelte
Comment (djbarnwal) [inline web-common/src/features/dashboards/dimension-table/ExportDimensionTableDataButton.svelte:126]: We should move this object inside script and reference here through a variable

File: web-common/src/features/dashboards/scheduled-reports/time-utils.ts
Comment (djbarnwal) [inline web-common/src/features/dashboards/scheduled-reports/time-utils.ts:12]: I would switch to using Luxon for most of these cases https://moment.github.io/luxon/api-docs/index.html#infoweekdays

File: web-common/src/features/dashboards/scheduled-reports/time-utils.ts
Comment (djbarnwal) [inline web-common/src/features/dashboards/scheduled-reports/time-utils.ts:24]: We have transformation methods present in `time` lib. Also luxon can be directly used as well

File: web-common/src/features/dashboards/time-series/MetricsTimeSeriesCharts.svelte
Comment (djbarnwal) [inline web-common/src/features/dashboards/time-series/MetricsTimeSeriesCharts.svelte:203]: Scrolling the charts container is buggy. The top axis should be always clearly visible when scrolling the container.
image

File: web-common/src/features/dashboards/scheduled-reports/time-utils.ts
Comment (djbarnwal) [inline web-common/src/features/dashboards/scheduled-reports/time-utils.ts:30]: I would look to check if this can already be done using luxon or Intl APIs

**Review by djbarnwal — APPROVED** [review]
Looks good. Have left some comments on using existing methods and libraries for handling time.

Also there is bug in the charts container on scrolling.

Approving so that you can merge after resolving the bug. I would leave it to you on how and when you want to address the other points.

File: web-common/src/components/forms/DatePicker.svelte
Comment (ericpgreen2) [reply web-common/src/components/forms/DatePicker.svelte]: Yeah, I'd like to, but our current `LitePicker` interface only supports selecting date ranges, not individual dates. This component was a quick stop-gap solution.

Anyways, I'm not actually using this component anywhere yet, so I've just removed it for now.

File: web-common/src/components/menu-v2/MenuItem.svelte
Comment (ericpgreen2) [reply web-common/src/components/menu-v2/MenuItem.svelte]: Actually I wasn't using this prop anywhere, so just removed it.

File: web-common/src/features/dashboards/scheduled-reports/time-utils.ts
Comment (ericpgreen2) [reply web-common/src/features/dashboards/scheduled-reports/time-utils.ts]: Actually an unused function. Removed.

**Pull request merged**
